res/eg/convert_to_script.sh

```
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# usage:
+#  ./res/eg/convert_to_script.sh res/eg/1d/jupyter/weak/weak.ipynb
+#
+
+set -ex
+
+jupyter nbconvert --to script $1 --output run
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)